### PR TITLE
Andy20160103

### PR DESCRIPTION
**Pull request by andreburto**
Added the "admin" check box on the user edit (by admin) page. Created the hasAdmin function and commented out the hasAuthorization function in users.authorization.server.controller.js file. Tested this with an admin user to change user information: it worked. Tested this with a non-admin user to change information: the red bar told me I wasn't an admin.

### CODE REVIEW

File: app/controllers/users/users.authorization.server.controller.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ exports.userByID = function(req, res, next, id) {
  * Require login routing middleware
  */
 exports.requiresLogin = function(req, res, next) {
+	console.log('requiresLogin');
```
Comment (anthonycrumley) [inline app/controllers/users/users.authorization.server.controller.js]: @andreburto Please remove the debug code.

**Comment by andreburto** [comment]
@anthonycrumley Removed.

**Comment by anthonycrumley** [comment]
This is implements #12.

**Comment by andreburto** [comment]
This latest commit resolves issue #148

File: app/controllers/users/users.profile.server.controller.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ exports.update = function(req, res) {
 	var message = null;
 
 	// For security measurement we remove the roles from the req.body object
-	delete req.body.roles;
+	// delete req.body.roles;
```
Comment (anthonycrumley) [inline app/controllers/users/users.profile.server.controller.js]: Please go ahead and delete these comments about role deletion.

File: public/modules/users/controllers/users.client.controller.js [hunk 3]
```diff
@@ -64,6 +65,44 @@ angular.module('users').controller('UsersController', ['$scope', '$stateParams',
...
+			
+			$scope.checkAdmin();
+		};
+		
+		// Add or remove the admin
+		$scope.setAdmin = function() {
+			if ($scope.isadmin === true) {
+				$scope.user.roles.push('admin');
+			} else {
+				while($scope.user.roles.indexOf('admin') > -1) {
```
Comment (anthonycrumley) [inline public/modules/users/controllers/users.client.controller.js]: Consider a loop like this...
```
while {
  var idx = $scope.user.roles.indexOf('admin');
  if (idx === -1) break;
  $scope.user.roles.splice(idx, 1);
}
```

**Comment by andreburto** [comment]
If I fix the tests to complete as-is it won't test for everything the current tests look at. An "admin" set of tests may be in order.

**Comment by andreburto** [comment]
Need to create nonadmin tests. Right now all tests that save data pass because the user is admin. This will require setting up one more user in the test. Will work on it asap.

**Comment by andreburto** [comment]
Two nonadmin tests pass: cannot create user and cannot update user.

**Pull request closed without merging**